usersession/userd/launcher.go

```diff
@@ -57,7 +57,7 @@ const launcherIntrospectionXML = `
 </interface>`
 
 var (
-	allowedURLSchemes = []string{"http", "https", "mailto", "snap", "help", "apt"}
+	allowedURLSchemes = []string{"http", "https", "mailto", "snap", "help", "apt", "zoommtg"}
```

Comment:
What kind of URLs are valid for zoom? Is it just http-style URLs or are there other things, like files?

Reply:
I think it's just http urls, e.g.:

```
zoommtg://mycompany.zoom.us/join?action=join&confid=<196 alphanumeric characters here>&confno=123456789&zc=0&pk=&mcv=0.92.11227.0929&browser=chrome
```